webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/MockServices.scala

```diff
@@ -310,12 +310,12 @@ class MockDirectives(mockTechniques: MockTechniques) {
     val commonTechnique = techniqueRepos.unsafeGet(TechniqueId(TechniqueName("common"), TV("1.0")))
     def commonVariables(nodeId: NodeId, allNodeInfos: Map[NodeId, NodeInfo]) = {
        val spec = commonTechnique.getAllVariableSpecs.map(s => (s.name, s)).toMap
-       Seq(
-         spec("OWNER").toVariable(Seq(allNodeInfos(nodeId).localAdministratorAccountName))
-       , spec("UUID").toVariable(Seq(nodeId.value))
-       , spec("POLICYSERVER_ID").toVariable(Seq(allNodeInfos(nodeId).policyServerId.value))
-       , spec("POLICYSERVER").toVariable(Seq(allNodeInfos(allNodeInfos(nodeId).policyServerId).hostname))
-       , spec("POLICYSERVER_ADMIN").toVariable(Seq(allNodeInfos(allNodeInfos(nodeId).policyServerId).localAdministratorAccountName))
+         Seq(
+           spec("OWNER").toVariable(Seq(allNodeInfos(nodeId).localAdministratorAccountName))
```

Comment:
this looks a lot like what is in NodeConfigData - i see that it's not easy to refactor the code to use the same source for both, but can you add a comment that state that NodeConfigData uses the same type of data and any change here is likely to need to be done on NodeConfigData as well ?

Reply:
for now, I didn't found a way to unify the two, but yes, it's horrible that they are not merged in some way. I'm adding cross-comments !